.env/.env.development

```diff
@@ -10,6 +10,8 @@ REACT_APP_ETH_REGISTRY_ADDRESS=0xBBeB72177EF2c0ec9fF96453f924a2b9590DF9B2
 REACT_APP_ETH_PROVIDER_URL=https://ropsten.infura.io/v3/c569c6faf4f14d15a49d0044e7ddd668
 REACT_APP_ETH_TOKEN_ADDRESS=0x65ca91574256B2c194A964E340e8B0A802E17ec6
 REACT_APP_ETH_OWNER_WALLET=0xcccc7428648c4AdC0ae262D3547584dDAE25c465
+COM_STOCK_URL=
```

Comment:
Nit -- this should be one word according to the nomenclature from dheeraj 
https://bioshock.fandom.com/wiki/Zachary_Hale_Comstock